workflowhelpers/internal/cf_auth_test.go

```diff
@@ -36,7 +37,7 @@ var _ = Describe("cf auth", func() {
 
 	Describe("CfAuth", func() {
 		It("runs the cf auth command", func() {
-			err := CfAuth(cmdStarter, redactingReporter, "user", password, 5*time.Second)
+			err := CfAuth(cmdStarter, redactingReporter, "user", password, "", 5*time.Second)
```

Comment:
Please use origin here as well to keep it consistent

Reply:
Done!